### PR TITLE
bitnami/all Fix ref links (in comments)

### LINKED ISSUE #17782: docs: links to https://kubernetes.io/docs/user-guide/...  end in 404 error
### Name and Version

bitnami/all

### What architecture are you using?

None

### What steps will reproduce the bug?

View a YAML files that includes a hyperlink, similar to the following:
```YAML
    ## ref: https://kubernetes.io/docs/user-guide/images/#pre-pulling-images
```

### Are you using any custom parameters or values?

_No response_

### What is the expected behavior?

Following a link loads the intended/expected content from kubernetes.io

### What do you see instead?

HTTP 404 error message from https://kubernetes.io/docs/ site

### Additional information

I intend to submit an MR to replace all ref links that include a link like `ref: https://kubernetes.io/docs/user-guide/` with a current/functional link

### PR DESCRIPTION
**Pull request by bcdady**
### Description of the change

Replace dead ref links, in values.yaml files, with live links

### Benefits

Provides accurate reference for users of charts

### Applicable issues

- fixes #17782 

### Additional information

Dead links includes http[s]://kubernetes.io/docs/user-guide/ in the path, and end in the browser showing the 404 page from <https://kubernetes.io/>

Replacement links typically reference /concepts/ in the path instead, and result in the browser showing the intended, relevant reference information.

### Checklist

- [ ] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by bcdady** [comment]
FYI This PR resumes where [#18799](https://github.com/bitnami/charts/pull/18799) left off.

**Comment by bcdady** [comment]
Hi @joancafom I see the CI job for checking the chart versions is failing

```shell
Error: Detected changes in charts without version bump in Chart.yaml. Charts changed: 106. Version bumps detected: 0
```

Previously (see [here](https://github.com/bitnami/charts/pull/18799#issuecomment-1708180198)), @FraPazGal had agreed that these comment/link updates did not merit changes to the chart versions. Do you know if there is any way to bypass that CI check?

File: bitnami/harbor/values.yaml [hunk 1]
```diff
@@ -2287,6 +2282,505 @@ registry:
...
+  ## Harbor Notary service configuration
+  ##
+  service:
+    ## @param notary.service.ports.server Harbor Notary server service port
+    ## @param notary.service.ports.signer Harbor Notary signer service port
+    ##
+    ports:
+      server: 4443
+      signer: 7899
+
```
Comment (joancafom) [inline bitnami/harbor/values.yaml:2783]: Why are you adding this section here?

File: bitnami/kube-prometheus/crds/crd-alertmanagers.yaml
Comment (joancafom) [inline bitnami/kube-prometheus/crds/crd-alertmanagers.yaml:1]: You should not update CRDs comments. Their content is not directly managed by us, but just a replication of the upstream project.

File: bitnami/kube-prometheus/crds/crd-prometheusagents.yaml
Comment (joancafom) [inline bitnami/kube-prometheus/crds/crd-prometheusagents.yaml:1]: You should not update CRDs comments. Their content is not directly managed by us, but just a replication of the upstream project.

**Review by joancafom — CHANGES_REQUESTED** [review]
Thanks for the PR. I left you some comments as a review, could you please check them?

File: bitnami/harbor/values.yaml (hunk 1 above)
Comment (bcdady) [reply bitnami/harbor/values.yaml:2783]: Hi @joancafom This is likely an unintended result of a rebase. Thanks for catching this; I'll remove the new lines.

File: bitnami/kube-prometheus/crds/crd-prometheusagents.yaml
Comment (bcdady) [reply bitnami/kube-prometheus/crds/crd-prometheusagents.yaml:1]: ok, I'll revert these.

File: bitnami/harbor/values.yaml (hunk 1 above)
Comment (bcdady) [reply bitnami/harbor/values.yaml]: lines +2285 to +2783 have been removed.

File: bitnami/kube-prometheus/crds/crd-alertmanagers.yaml
Comment (bcdady) [reply bitnami/kube-prometheus/crds/crd-alertmanagers.yaml:1]: CRD changes have been dropped

File: bitnami/kube-prometheus/crds/crd-prometheusagents.yaml
Comment (bcdady) [reply bitnami/kube-prometheus/crds/crd-prometheusagents.yaml:1]: CRD changes dropped.

**Comment by joancafom** [comment]
Now it's the DCO check that's failing, could you please follow the instructions in https://github.com/bitnami/charts/pull/21822/checks?check_run_id=20273292093 to solve it? @bcdady

**Comment by bcdady** [comment]
Hi @joancafom Have you had any chance to re-review since my latest updates?

**Review by joancafom — APPROVED** [review]

**Comment by joancafom** [comment]
LGTM! 🚀

**Pull request merged**